src/cs-graphics/SetupGLNode.cpp

```diff
@@ -27,6 +77,9 @@ bool SetupGLNode::Do() {
   glEnable(GL_CULL_FACE);
   glCullFace(GL_BACK);
 
+  // Attach the debug callback to print the messages
+  glDebugMessageCallback(MessageCallback, static_cast<void*>(mSettings.get()));
```

Comment:
Isn't the `Do` function part of the frame loop and called every frame? This function has to be called only once at startup.